verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -717,34 +805,5 @@ std::ostream& operator<<(std::ostream& stream,
   return stream;
 }
 
-KytheIndexingData CollectKytheIndexingData(
```

Comment:
@fangism we'll need to change the internal verification tests. It depends on this function.

@MinaToma can you provide a function similar to this one? (a function that returns complete KytheIndexingData)

Reply:
Ack.  @ivan444 I'll ask you to handle the PR import for this one.  I can share instructions offline.